app/src/main/java/org/mozilla/tv/firefox/pocket/PocketVideoParser.kt

```diff
@@ -21,9 +21,24 @@ object PocketVideoParser {
             url = jsonObj.optString("tv_url", null) ?: jsonObj.getString("url"),
             thumbnailURL = jsonObj.getString("image_src"),
             popularitySortId = jsonObj.getInt("popularity_sort_id"),
-            authors = jsonObj.getString("authors")
+            authors = getAuthorName(jsonObj)
         )
     } catch (e: JSONException) {
         null
     }
+
+    private fun getAuthorName(jsonObj: JSONObject): String {
+        var authorName = ""
+        try {
+            val authorsJSON = JSONObject(jsonObj.getString("authors"))
+
+            for (x in authorsJSON.keys()) {
+                val authorObject = JSONObject(authorsJSON[x].toString())
+                authorName = authorObject.get("name").toString()
```

Comment:
If the JSON contains multiple authors, this will always return the last one.  Is that intended?  If so, we should rename the field on `FeedItem.Video` from `authors` -> `author`.  If not, `authorName` should probably be a list (and will need to be renamed).

Reply:
That was my intended behaviour, because I assumed there wouldn't be more than one author. But it would be more correct to handle multiple authors, I'll change that.

Reply:
Verifying in the issue if it is possible for there to be more than one author.